src/views/htmlsandbox.tsx

```diff
@@ -36,7 +36,7 @@ const HtmlSandbox: Component<HtmlSandboxProps> = (props) => {
         <Switch>
             <Match when={sanitizedHtml.state === "ready"}>
                 <div
-                    class="post-content overflow-auto break-words"
+                    class="post-content p-3 overflow-auto break-words"
```

Comment:
Instead of defining padding on `HtmlSandbox` itself, which is used in multiple places, I would prefer to have each place that uses `HtmlSandbox` be able to optionally set padding (and maybe other classes) on the `HtmlSandbox` component.

I think probably the cleanest way to accomplish that is to set up something similar to how `splitProps` is used in `PostBody` and `CardContent`.

Reply:
still learning the ropes of solidjs, so I don't understand props much just yet. i ended up adding a wrapper around the htmlsandbox element in the postbody where I could add the padding in the meantime, hopefully that's alright for now.